app/Helpers/database/user.php

```diff
@@ -61,6 +61,7 @@ function getUserPageInfo(string $username, int $numGames = 0, int $numRecentAchi
     $libraryOut['MemberSince'] = $user->created_at->__toString();
     $libraryOut['LastActivity'] = $user->LastLogin?->__toString();
     $libraryOut['RichPresenceMsg'] = empty($user->RichPresenceMsg) || $user->RichPresenceMsg === 'Unknown' ? null : $user->RichPresenceMsg;
+    $libraryOut['RichPresenceUpdated'] = $user->RichPresenceMsgDate?->__toString();
```

Comment:
nit: For consistency with `RichPresenceMsg`, let's keep this new field name similarly aligned with the DB column: `RichPresenceMsgDate`.